OrangeHRMTests/Common/WebElements/MyWebElement.cs

```diff
@@ -0,0 +1,103 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.Drivers;
+using OrangeHRMTests.Common.Extensions;
+using System.Collections.ObjectModel;
+using System.Drawing;
+using OpenQA.Selenium.Support.UI;
+
+namespace OrangeHRMTests.Common.WebElements
+{
+    public class MyWebElement : IWebElement
+    {
+        protected By By { get; set; }
+
+        protected IWebElement WebElement => WebDriverFactory.Driver.GetWebElementWhenExist(By);
+
+        public string TagName => WebElement.TagName;
+
+        public string Text
+        {
+            set { Waiter(); }
```

Comment:
waiter is a method that wait for disappearing specific element, but it's named like an object

Reply:
Fixed